### PR TITLE
Basic structure for SFTP transport with happy path

### PR DESCRIPTION
**Pull request by mry-usds**
This PR does defines a basic structure for the Transport/Send Azure functions and has a happy-path implementation for SFTP

## Changes
- Adds the azure function "Send" which waits for blobs placed into the processed container
- Uses the blob to send (default) SFTP to a localhost location/service
- Establishes the placement of metadata into the receivers.yml file and starts to build a structure for that
- Has placeholders and a start to dynamically calling the correct Transport from the metadata

## Checklist
- [X] Tested locally?
- [X] Added new dependencies?
- [ ] Updated the docs?
- [ ] Added a test?
- [ ] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 

- #issue

## To Be Done
- Add tests
- #issue

### CODE REVIEW

File: prime-router/pom.xml [hunk 1]
```diff
@@ -166,7 +166,7 @@
                     <archive>
                         <manifest>
                             <addClasspath>true</addClasspath>
-                            <classpathPrefix>${stagingDirectory}/lib</classpathPrefix>
+                            <classpathPrefix>lib/</classpathPrefix>
```
Comment (RickHawesUSDS) [inline prime-router/pom.xml:169]: Not sure about this change. It breaks what I setup for the prime script. Why?

File: prime-router/src/main/kotlin/Receiver.kt [hunk 2]
```diff
@@ -24,6 +27,20 @@ data class Receiver(
         }
     }
 
+    data class Transport(
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/Receiver.kt:30]: +1 on this class.

File: prime-router/src/main/kotlin/azure/SendFunction.kt [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    fun run(
+        @BlobTrigger( name = "content", path="%PROCESSED_BLOB_CONTAINER%/{fileName}.csv", dataType = "binary") content: ByteArray,
+        @BindingName("fileName") fileName: String,
+        context: ExecutionContext
+    ) {
+        val baseDir = System.getenv("AzureWebJobsScriptRoot")
+        Metadata.loadAll("$baseDir/metadata")
+
+        context.logger.info("Dispatch function processed a blob. Name: ${fileName} Size: ${content.size} bytes");
+
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/azure/SendFunction.kt:40]: +1 on logging info about the payload

**Review by RickHawesUSDS — CHANGES_REQUESTED** [review]
One change in the POM that I'm requesting. Also, is there a docker SFTP server that we should be running? Should that be put into the docker-compose.yml file?

File: prime-router/pom.xml (hunk 1 above)
Comment (mry-usds) [reply prime-router/pom.xml:169]: thats a hold over from the older issues i was having with Windows...didnt realize that was in there...still

**Review by RickHawesUSDS — APPROVED** [review]

**Pull request merged**